corehq/ex-submodules/casexml/apps/phone/tasks.py

```diff
@@ -46,12 +46,9 @@ def get_async_restore_payload(restore_config, domain=None, username=None):
     Process an async restore
     domain and username: added for displaying restore request details on flower
     """
-    try:
-        repr(restore_config)
-    except Exception as e:
-        logging.error('Something went wrong with RestoreConfig.__repr__() : {msg}'.format(
-            msg=str(e)
-        ))
+    logging.info('RestoreConfig in get_async_restore_payload task: {msg}'.format(
```

Comment:
```suggestion
    log.info('RestoreConfig in get_async_restore_payload task: {msg}'.format(
```

Also somewhere near the top of the file just below the imports you'll need to add
```py
log = logging.getLogger(__name__)
```